internal/apitest/config.go

```diff
@@ -33,7 +33,7 @@ type Suite struct {
 	Timeout        time.Duration     `yaml:"timeout,omitempty"`
 	Name           string            `yaml:"name,omitempty"`
 	ProjectName    string            `yaml:"projectName,omitempty"`
-	HookID         string            `yaml:"hookId,omitempty"`
+	HookID         string            `yaml:"-"`
```

Comment:
Do we still need to keep this field in the struct?

Reply:
This is the one that will be used to actually start the job.
Since the translation `ProjectName` > `HookID` is done before running the project, it's more convenient to keep it there rather that introduce a new object that will be exactly the same as config except that field in addition.

I hope in the future it can be possible to invoke directly through a project name.

Reply:
Ah OK! It might be good to visually separate the `HookID` field in the struct from the other fields and perhaps add a doc that explains where this comes from 🙏

Reply:
I updated to add a more complete description !